### PR TITLE
Weapon Tweaks - Ballistic Penetration & X-Ray Firing Speed

### PR DESCRIPTION
**Pull request by WickedCybs**
This PR sets out to do a few things. Looking at rifle penetration and laser weapon firing speed mainly.

For rifles, 5.56 has been set to not penetrate through things in the interest of balance, the AP variant still can. 7.62 is unchanged for now. 

X-ray weapons had a case where a few of them could fire as fast as the user could click and this actually seemed intentional as the firedelay was specifically set to one on the carbine. I noticed this with the x-rays and nerfed them so that they have a longer delay. The lack of it meant easily frying people with high AP lasers. The carbine still fires faster than a rifle can.

Also, the submachine gun turret was firing rifle rounds, I made it shoot pistol medium instead, having it be more of a .45 submachine gun.

### CODE REVIEW

**Comment by WickedCybs** [comment]
!review

**Review by JohnWildkins — APPROVED** [review]
something something technically good maintainer final decision green checkmark

**Review by Geevies — APPROVED** [review]

**Pull request merged**
